packages/demo/src/components/examples/ChildFormExamples.tsx

```diff
@@ -34,7 +34,7 @@ const SimpleChildFormExample: React.FunctionComponent = () => {
                     </ChildForm>

```

Comment:
ChildForm have lost their background color
Before:
![image](https://user-images.githubusercontent.com/35579930/107647468-30ad2c80-6c49-11eb-9a57-6533a522b1ed.png)

After:
![image](https://user-images.githubusercontent.com/35579930/107647399-1ecb8980-6c49-11eb-9fad-a12a5ef42687.png)

Reply:
I'll take a look, it's probably my changes :eyes: